### PR TITLE
EOBS 1117, 1260, 1233, 1237, 283, 1272, 1273, 1274, 1276, 1281, 1283

### PR DESCRIPTION
**Pull request by technotariat**
- Adding test to prove that PME isn't shown in report that has date range within PME date range
- Changing domain used to get PMEs for reports
- Changing verification methods
- Fixing flaky test

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
LGTM as long as Travis and Codacy go green :shipit:

File: nh_eobs_mental_health/tests/report_nh_clinical_observation_report/test_get_patient_monitoring_exception_report_data_obs_stop.py [hunk 1]
```diff
@@ -94,161 +94,88 @@ def setUp(self):
...
 
-        # Setup cancellation of patient monitoring exception due to transfer.
-        cancel_reason_transfer = \
-            self.browse_ref('nh_eobs.cancel_reason_transfer')
-        self.pme_cancelled_due_to_transfer_activity.cancel_reason_id = \
-            cancel_reason_transfer.id
+    def call_test(self, expected_num_stop_obs_entries,
+                  expected_num_restart_obs_entries,
+                  expected_reason_display_text='',
+                  report_start_datetime = None, report_end_datetime = None):
```
Comment (moderatemisbehaviour) [inline nh_eobs_mental_health/tests/report_nh_clinical_observation_report/test_get_patient_monitoring_exception_report_data_obs_stop.py]: Add docstring.

File: nh_eobs_mental_health/tests/report_nh_clinical_observation_report/test_get_patient_monitoring_exception_report_data_obs_stop.py [hunk 2]
```diff
@@ -263,144 +190,168 @@ def test_returns_dictionary(self):
...
         """
-        pme_ended_report_entries = self.get_restart_obs_report_entries()
+        reason_display_text = 'pme_open_reason'
+        self.setup_pme(reason_display_text)
+        self.call_test(1, 0, reason_display_text)
 
-        self.assertEqual(len(pme_ended_report_entries), 4,
-                         "Unexpected number of 'ended' report entries.")
+    def test_pme_started_long_ago(self):
+        reason_display_text = 'pme_started_long_ago_reason'
```
Comment (moderatemisbehaviour) [inline nh_eobs_mental_health/tests/report_nh_clinical_observation_report/test_get_patient_monitoring_exception_report_data_obs_stop.py]: Add docstrings.

**Comment by Nayira** [comment]
Test for Bugs 1117, 283, 1259, 1260, Change request 1237 and Feature branch bugs 1233, 1272, 1273, 1274 and 1276 are a pass. :shipit:

**Pull request merged**